### PR TITLE
Add link button to discovery.

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: templates/DataDiscovery/index.html.twig [hunk 1]
```diff
@@ -85,6 +85,9 @@
                             <td valign="top">
                                 <input type="button" id="clear-button" class="button" value="Clear/Show All" onclick="clearAll();">
                             </td>
+                            <td valign="top">
+                                <input type="button" id="searchlink-button" class="button" value="Search" onclick="parent.location=Routing.generate('pelagos_app_ui_searchpage_default')">
```
Comment (mickel1138) [inline templates/DataDiscovery/index.html.twig:89]: It's say the onClick is a nono. If you are going to use jQuery, generate the link in javascript.

Comment (mickel1138) [reply templates/DataDiscovery/index.html.twig:89]: And also would use window, rather than parent.

File: assets/static/css/data-discovery.css [hunk 2]
```diff
@@ -241,3 +241,22 @@ input.ui-button {
 #expand-collapse.collapsed {
     left: -12px;
 }
+
+.teal-button
+{
+    background-image:none;
+    background-color:darkcyan;
```
Comment (mickel1138) [inline assets/static/css/data-discovery.css:248]: This color should be: `var(--color-headerMiddle)`
It's a bad name, but it's the right one.

If you don't this won't work in the other template.

File: assets/static/css/data-discovery.css [hunk 3]
```diff
@@ -241,3 +241,22 @@ input.ui-button {
...
+    color:white;
+}
+.teal-button:hover
```
Comment (mickel1138) [inline assets/static/css/data-discovery.css:251]: If you made no changes, why is there a hover, and active?

File: templates/DataDiscovery/index.html.twig [hunk 4]
```diff
@@ -85,6 +85,9 @@
                             <td valign="top">
                                 <input type="button" id="clear-button" class="button" value="Clear/Show All" onclick="clearAll();">
                             </td>
+                            <td valign="top">
+                                <input type="button" id="searchlink-button" class="button teal-button" value="Search V2" onclick="navigateToNewsearch();">
```
Comment (mickel1138) [inline templates/DataDiscovery/index.html.twig:89]: Still has onClick, use jQuery.on, or if you wanna be cool use: `document.getElementById("searchlink-button").addEventListener("click", navigateToNewsearch);`

Comment (mickel1138) [reply templates/DataDiscovery/index.html.twig:89]: Can even use an anonymous function in the event listener.

File: assets/static/css/data-discovery.css (hunk 3 above)
Comment (fightingtexasaggie) [reply assets/static/css/data-discovery.css:251]: hover was still needed, active not

File: assets/static/css/data-discovery.css (hunk 2 above)
Comment (fightingtexasaggie) [reply assets/static/css/data-discovery.css:248]: ok, works as suggested.

File: assets/static/css/data-discovery.css (hunk 3 above)
Comment (mickel1138) [reply assets/static/css/data-discovery.css:252]: I was mistaken, and active can even be used. But the point was that the color didn't change, so you can't see that you are hovering. Which may be ok.

**Pull request merged**